butterknife-annotations/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'java'
+apply plugin: 'checkstyle'
+
+targetCompatibility = JavaVersion.VERSION_1_7
+sourceCompatibility = JavaVersion.VERSION_1_7
+
+checkstyle {
+  configFile rootProject.file('checkstyle.xml')
+  showViolations true
+}
+
+dependencies {
+  compile 'com.google.android:android:4.1.1.4'
```

Comment:
What is this needed for? It should definitely not be `compile` scope if needed but `provided`.